src/blog/multiPage.html

```diff
@@ -95,7 +95,7 @@
               </div>
             </div> <!-- end col -->
 
-            <div class="menu-socials hidden-sm hidden-xs">
+            <div class="menu-socials hidden-sm hidden-xs" style="display:none;">
```

Comment:
- If we may implement these blocks of code --> comment out
- If we won't be using them --> remove from codebase

...tbd?